blog/2023-10-02-kuzu-v-0.0.9.md

```diff
@@ -62,32 +65,34 @@ CALL SHOW_TABLES() RETURN *;
 ----------------------------------
 ```
 
-### Recursive Rel Projection
-This release expand recursive rel pattern and enables projection on intermediate nodes and rels. Previously, Kùzu only supports returning all node and rel properties on the path.
+### Recursive Relationship Projection
+This release expands recursive relationship patterns and enables projection on intermediate nodes and relationships. Previously, Kùzu only supported returning all node and relationship properties on the path.
 ```
-MATCH (a:User)-[e:Follows*1..2 (r, n | WHERE r.since > 2020)]->(b:User) 
+MATCH (a:User)-[e:Follows*1..2 (r, n | WHERE r.since > 2020)]->(b:User)
 RETURN nodes(e), rels(e);
 ```
-This approach incurs significant computation overhead when user only interested in a subset of properties over path. Also returning all properties make the result hard to interpret.
+This incurs significant computation overhead when a user is only interested in a subset of properties on the path. Also, returning all properties makes the result harder to interpret.
 
-Kùzu now allow projection inside recursive rel pattern using a list-comprehension-like syntax.
+Kùzu now allow projection inside recursive relationship patterns using a list-comprehension-like syntax.
 ```
-MATCH (a:User)-[e:Follows*1..2 (r, n | WHERE r.since > 2020 | {r.since}, {n.name})]->(b:User) 
+MATCH (a:User)-[e:Follows*1..2 (r, n | WHERE r.since > 2020 | {r.since}, {n.name})]->(b:User)
 RETURN nodes(e), rels(e);
 ```
-The query above finds all 1 to 2 hops Follows path since 2020. The project `since` property of intermediate rels and `name` property of intermediate nodes along the path.
+The query above finds all paths between two users which is between 1 to 2 hops, and where the users have followed eachother after 2020. The query returns the `since` property of any `Follow` relationships and the name of any intermediate users.
```

Comment:
```suggestion
The query above finds all paths between two users which are between 1 and 2 hops, and where the users followed each other after 2020. The query returns the `since` property of any `Follow` relationships and the name of any intermediate users.
```